src/docs/foundation/accessibility.mdx

```diff
@@ -0,0 +1,59 @@
+---
+name: Accessibility
+menu: Foundation
+route: /foundation/accessibility
+---
+
+# Accessibility
+
+React UI bakes accessibility principles right into its core.
+
+## Touch Friendliness
+
+The active area of interactive elements should be properly sized so that the
+elements can be easily targeted on touch screens. Recommended dimensions may
+vary from platform to platform, however a commonly used size is 7–10 mm.
+
+Default tap target size in React UI is set to **10 mm** and is used by all
+potentially small interactive components like [Alert](/components/ui/alert)
+close button, [CheckboxField](/components/ui/checkbox-field), or
+[Toggle](/components/ui/toggle). Tap target size can be adjusted via the
+`--rui-tap-target-size` custom property (see
+[Theming](/customize/theming/overview) to learn how).
+
+📖 [Read more about touch targets at Norman Nielsen Group.](https://www.nngroup.com/articles/touch-target-size/)
+
+### Form Fields and Reserved Space
+
+Note that form fields with potentially small inputs (like
+[CheckboxField](/components/ui/checkbox-field) or
+[Toggle](/components/ui/toggle)) reserve vertical space corresponding to the
+minimum tap target size. In other words, form fields **box model is taller.**
+The reason behind this behaviour is that in many cases the minimum tap target
+size could overflow its component's box model and tap targets of neighboring
+components could collide. The extra added space prevents this.
+
+However, if placed inside [FormLayout](/components/layout/form-layout), form
+fields do not add any extra vertical space because it is already provided by
+`FormLayout` row gap. Remember to check that form fields in your `FormLayout`
+are properly spaced and interactive elements do not collide should you decide to
+make any changes to `--rui-tap-target-size`,
```

Comment:
```
and interactive elements do not collide should you decide to
 make any changes to `--rui-tap-target-size`,
 `--rui-form-field-check-tap-target-size` or `--rui-form-layout-row-gap` options.
```

My English skills are not at the highest levels, so I am not sure about it. However, this part of sentence seems weird to me, like something is missing here. I

Reply:
I'm afraid my English is rather disastrous, but if you mean the phrase that goes "should you decide", it's the same as "if you (should) decide", just a bit more formal.

https://forum.thefreedictionary.com/postst12144_Should-you-have-any-questions-If-you-have-any-questions.aspx